src/main/java/frc/robot/commands/SwerveDriveCommands/DriveConstantControlCommand.java

```diff
@@ -1,98 +1,105 @@
 package frc.robot.commands.SwerveDriveCommands;
 
+import static frc.robot.Constants.DriveConstants.maxAngularSpeed;
+import static frc.robot.Constants.DriveConstants.maxSpeed;
+
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.subsystems.swerve.DetermineDriveSpeed;
+import frc.robot.subsystems.swerve.DriveSpeed;
 import frc.robot.subsystems.swerve.SwerveDrive;
-
-import static frc.robot.Constants.AutoConstants;
-import static frc.robot.Constants.OIConstants;
+//import frc.robot.subsystems.swerve.SwerveModule;
+import frc.robot.Constants.*;
 
 public class DriveConstantControlCommand extends CommandBase {
 
-  private final SwerveDrive m_swerveDrive;
-  private final XboxController controller;
-  private final DetermineDriveSpeed limJoystickLeft = new DetermineDriveSpeed(0.05);
-  private final DetermineDriveSpeed limJoystickRight = new DetermineDriveSpeed(0.05);
+    private final SwerveDrive m_swerveDrive;
+    private XboxController controller;
+    private final DriveSpeed limJoystickLeft = new DriveSpeed(0.05);
+    private final DriveSpeed limJoystickRight = new DriveSpeed(0.05);
+    public DriveConstantControlCommand(SwerveDrive swerveDrive, XboxController controller) {
+        m_swerveDrive = swerveDrive;
+        this.controller = controller;
+        addRequirements(swerveDrive);
+    }
 
-  public DriveConstantControlCommand(SwerveDrive swerveDrive, XboxController controller) {
-    m_swerveDrive = swerveDrive;
-    this.controller = controller;
 
-    addRequirements(swerveDrive);
-  }
 
-  @Override
-  public void initialize() {}
+    @Override
+    public void initialize() {}
 
-  // Called every time the scheduler runs while the command is scheduled.
-  @Override
-  public void execute() {
-    double leftX, leftY, rightX;
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+        double leftX, leftY, rightX;
 
-    leftX = controller.getLeftX();
+    leftX = -controller.getLeftX();
     leftY = -controller.getLeftY();
     rightX = controller.getRightX();
 
-    // Find radii for controller dead-zones (circular)
-    double leftRadius = Math.sqrt(Math.pow(leftX, 2) + Math.pow(leftY, 2));
-    double rightRadius = Math.abs(rightX);
 
-    // add input curves
-    leftX = Math.pow(leftX, 3);
-    leftY = Math.pow(leftY, 3);
-    rightX = Math.pow(rightX, 3);
+        // Find radii for controller dead-zones (circular)
+        double leftRadius = Math.sqrt(Math.pow(leftX, 2) + Math.pow(leftY, 2));
+        double rightRadius = Math.abs(rightX);
+
+//        // add input curves
+//        leftX = Math.pow(leftX, 3);
+//        leftY = Math.pow(leftY, 3);
+//        rightX = Math.pow(rightX, 3);
```

Comment:
We should delete this code. A better version of this logic is on the arm branch as a helper method: https://github.com/WestwoodRobotics/FRC-2023/blob/transport/src/main/java/frc/robot/util/Conversions.java#L88-L95